### PR TITLE
Add immunoAssay keys

### PR DESCRIPTION
**Pull request by cw-dvr8**
Add keys to the immunoAssay module for AMP.

### CODE REVIEW

File: terms/immunoAssays/antibodyDilution.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "sage.annotations-immunoAssays.antibodyDilution-0.0.1",
+    "description": "Ratio of antibody used in an assay",
+    "type": "string"
```
Comment (karawoo) [inline terms/immunoAssays/antibodyDilution.json:5]: Will people be providing this as a fraction, not a decimal?

Comment (cw-dvr8) [reply terms/immunoAssays/antibodyDilution.json:5]: @Aryllen would you comment on this?

Comment (Aryllen) [reply terms/immunoAssays/antibodyDilution.json:5]: That's a good question, @karawoo. This is expected to be a ratio like 10:100. If I remember correctly from the conversation when this was being discussed, the researchers tended to use the ratio. Should this be more clear?

Comment (karawoo) [reply terms/immunoAssays/antibodyDilution.json:5]: I think it's fine if it will be familiar to the researchers

Comment (Aryllen) [reply terms/immunoAssays/antibodyDilution.json:5]: Cool! Thank you for questioning!

**Review by Aryllen — APPROVED** [review]
Thank you, @cw-dvr8!

**Pull request merged**